### DISCUSSION TITLE
Woche 06 - Metriken

### DISCUSSION THREAD

**Discussion by lukashert** [Chrissi-Ruege/HealthKeeper#29] [General]
# Metriken
In dieser Woche haben wir uns mit folgenden Software Metriken beschäftigt. 

## Maintainability Index: 86 von 100
Unser System erreicht einen Maintainability Index von 86 Punkten auf einer Skala von maximal 100. Dieser hohe Wert deutet darauf hin, dass unser Code sehr gut wartbar ist. Ein hoher Maintainability Index erleichtert es Entwicklern, Änderungen und Erweiterungen vorzunehmen, ohne unbeabsichtigte Fehler einzuführen. Dies spart nicht nur Zeit, sondern reduziert auch die langfristigen Wartungskosten.

## Cyclomatic Complexity: 242
Die Cyclomatic Complexity unseres Systems liegt bei 242. Diese Metrik misst die Anzahl der linearen unabhängigen Pfade durch den Code. Ein höherer Wert deutet auf eine höhere Komplexität hin, was die Testbarkeit und das Verständnis des Codes erschwert.

## Class Coupling Score: 125
Der Class Coupling Score unseres Systems beträgt 125. Class Coupling misst, wie stark die Klassen in einem System miteinander verbunden sind. Ein niedrigerer Wert ist wünschenswert, da ein geringer Coupling-Grad die Flexibilität und Wiederverwendbarkeit des Codes erhöht. Ein Wert von 125 ist moderat.

**Comment by niklaskrauth** [comment]
Es ist schön zu sehen das euer Score so weit oben ist.
